packages/api/src/scripts/import-praise.ts

```diff
@@ -100,7 +100,7 @@ const importPraise = async (
   process.exit();
 };
 
-connectDatabase().then(() => {
+connectDatabase('localhost').then(() => {
```

Comment:
Please see my changes to this file in #289 -- I think it is a cleaner approach to per-environment overriding of env variables.

Reply:
👍 ok great, any action needed on my part?

Reply:
Yeah `connectDatabase` no longer takes any parameters - we're getting build errors due to calling it with a single parameter: `connectDatabase('localhost')`.

But also #289 changes the connectDatabase implementation -- we may need to carefully merge.